### PR TITLE
Update add delete and mark assignment command

### PR DESCRIPTION
**Pull request by qinguorui2001**

### CODE REVIEW

**Comment by weichang18** [comment]
I think your portion and mine has alot of conflicts in our implementation. However, when you are deconflicting in order to merge, just try to include both sides(comment it out if needed), so I can decide on how to integrate it later

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -98,6 +104,40 @@ public void removePerson(Person key) {
...
+    /**
+     * Adds an assignment to the person's assignment list.
+     * The assignment must not already exist in the person's assignment list.
+     */
+    public void addAssignment(Name name, Assignment assignment) {
+        persons.personWithSameName(name).getAssignments().add(assignment);
+    }
+
+    /**
+     * Removes {@code key} from this {@code AddressBook's person assignment list}.
```
Comment (TaoJun99) [inline src/main/java/seedu/address/model/AddressBook.java:126]: I think "{@code AddressBook's person assignment list}" should be "{@code List<Assignment>}" or "{@code UniqueAssignmentList}" based on the implementation

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -98,6 +104,40 @@ public void removePerson(Person key) {
...
+     * {@code key} must exist in the person assignment list.
+     */
+    public void removeAssignment(Name name, Assignment key) {
+        persons.personWithSameName(name).getAssignments().delete(key);
+    }
+
+    /**
+     * Marks {@code key} from this {@code AddressBook's person assignment list}.
```
Comment (TaoJun99) [inline src/main/java/seedu/address/model/AddressBook.java:134]: Same as above

**Review by TaoJun99 — COMMENTED** [review]
As Wei Chang mentioned, they may be some conflicts in the implementation. Other than that, code quality and everything else looks alright!

**Review by weichang18 — APPROVED** [review]

**Pull request merged**